Mergin/utils.py

```diff
@@ -240,12 +242,17 @@ def send_logs(username, logfile):
             username
         )
 
+    global_logs = "".encode()
+    if global_log_file:
+        with open(global_log_file, 'rb') as f:
+            global_logs = f.read() + "\n--------------------------------\n\n".encode()
```

Comment:
we could do something similar to what is below - if the size of global log is larger than e.g. 100kb, we would only take the last 100kb...